packages/plugins/other/visitor-plugin-common/src/base-types-visitor.ts

```diff
@@ -174,6 +176,22 @@ export interface RawTypesConfig extends RawConfig {
    * ```
    */
   onlyOperationTypes?: boolean;
+  /**
+   * @description This will cause the generator to use the implementing types of an interface as the type for a field.
```

Comment:
`When a GraphQL interface is used for a field, this flag will use the implementing types, instead of the interface itself.`